### PR TITLE
Fix Mapping tests by using regex

### PR DESCRIPTION
**Pull request by landim**
Changing the validation of generated queries using regex instead of string comparison.

### CODE REVIEW

**Comment by landim** [comment]
The insert tests have been fixed. I will move to fix update linq tests.

**Comment by landim** [comment]
The `AtomicMonotonicTimestampGenerator` tests are failing. Wdyt about use `Assert.GreaterThan` instead of `AreEquals` as the result can be unpredictable?

**Comment by jorgebay** [comment]
Nevermind the TimestampGenerator tests, we could fix them in a separate PR.

**Comment by landim** [comment]
I've rerun the tests and looks its ready for review!

File: src/Cassandra.Tests/Mapping/Linq/LinqToCqlUpdateUnitTests.cs [hunk 1]
```diff
@@ -69,8 +69,9 @@ public void Update_Multiple_Where_Test()
                 .Select(t => new AllTypesEntity { StringValue = "Billy the Vision" })
                 .Update()
                 .Execute();
-            Assert.AreEqual("UPDATE tbl1 SET string_val = ? WHERE id = ? AND val2 > ?", query);
-            CollectionAssert.AreEqual(new object[] { "Billy the Vision", id, 20M }, parameters);
+            TestHelper.VerifyUpdateCqlColumns(@"tbl1", query, new []{@"string_val"},
```
Comment (jorgebay) [inline src/Cassandra.Tests/Mapping/Linq/LinqToCqlUpdateUnitTests.cs]: No need for verbatim string literal, `@"something"`

**Comment by landim** [comment]
Addressed all the suggestions

**Comment by jorgebay** [comment]
that was fast!

**Comment by jorgebay** [comment]
lgtm! +1 :shipit:

**Pull request merged**